Doc/library/dbm.rst

```diff
@@ -344,26 +344,28 @@ to locate the appropriate header file to simplify building this module.
 
 --------------
 
-The :mod:`dbm.dumb` module provides a persistent dictionary-like interface which
-is written entirely in Python.  Unlike other modules such as :mod:`dbm.gnu` no
-external library is required.  As with other persistent mappings, the keys and
-values are always stored as bytes.
-
-The module defines the following:
+The :mod:`dbm.dumb` module provides a persistent :class:`dict`-like interface which
+is written entirely in Python.
+Unlike other :mod:`dbm` backends, such as :mod:`dbm.gnu` no,
+external library is required.
+As with other persistent :term:`mappings <mapping>`,
+the keys and values are always stored as :class:`bytes`.
```

Comment:
```suggestion
The :mod:`dbm.dumb` module provides a persistent :class:`dict`-like
interface which is written entirely in Python.
Unlike other :mod:`dbm` backends, such as :mod:`dbm.gnu` no
external library is required.
As with other persistent :term:`mappings <mapping>`,
the keys and values are always stored as :class:`bytes`.
```
Removed an extra comma and rewrapped.

I'm not sure the link to `mappings` is useful here, since it's talking about "persistent mappings", which I assume it refers to other `dbm` modules.  Even though they are "mapping-like", the link to mapping might end up being confusing since generally mappings are not persistent nor store they keys/values as bytes.

Reply:
We might want to word that slightly different, yes. Removing the existing link to mappings could help. Alternatively, mention instead the other dbm backends.

Reply:
I'll apply the reflow suggestion first, then I'll open a new conversation for adjusting the "mappings" part.